library/std/tests/switch-stdout.rs

```diff
@@ -1,10 +1,9 @@
-// run-pass
-// ignore-wasm (needs file descriptors and env variables)
+#[cfg(any(target_family = "unix", target_family = "windows"))]
```

Comment:
You meant to write `#!`

Reply:
Submitted a PR for this

Reply:
Why is this? I want to ignore all the other systems

Reply:
A `#[cfg(...)]` is a configuration specification for the next line. So in this case it says something about the `use std::fs::File;` (Yes the new line is confusing here, which is why it slipped through review.).
A `#![cfg(...)]` says something about the entire module. That's what you want. It's pure coincidence that this wasn't picked up by CI either.

Reply:
Oof, sorry for missing that.

Reply:
I'm also sorry 😕

Reply:
Writing software is hard. That's why we run things in CI :)